code/obj/item/implant.dm

```diff
@@ -460,36 +460,6 @@ THROWING DARTS
 				//src.on_remove(H)
 				//H.implant.Remove(src)
 				//src.set_loc(get_turf(H))
-			else if (H.mind in ticker.mode:revolutionaries)
-				H.TakeDamage("chest", 1, 1, 0)
-				H.changeStatus("weakened", 1 SECOND)
-				H.force_laydown_standup()
-				H.emote("scream")
-				playsound(H.loc, "sound/effects/electric_shock.ogg", 60, 0,0,pitch = 1.6)
```

Comment:
I'd say probably best to leave this bit, so it's still obvious to those without sechuds that you've implanted a rev and need to follow up.

Reply:
Revs already get a custom sechud status effect + if youre going to have counterrevs constantly damage them then why not have it automatically convert so you dont get a bunch of dead revs, which is exactly the change that this change is going against

Reply:
This is the initial hit effect, not the damage over time. Removing that means anyone without sechuds will have zero way of knowing if the person they just implanted is a rev or not and will be forced to beat them just to find out.

Reply:
I think thats good, it encourages them to buddy up with secoffs/play defensively instead of offensively (which helps the offensively and likely lower population revs win)